src/components/2FA/ActivationConfirmation.jsx

```diff
@@ -5,15 +5,19 @@ import { Media, Img, Bd } from 'cozy-ui/transpiled/react/deprecated/Media'
 
 import ReactMarkdownWrapper from 'components/ReactMarkdownWrapper'
 
-export const ActivationConfirmation = ({ images, error }) => {
+const twoFaModalBanner = require('assets/images/double_authent_prez_banner.svg')
+const twoFaModalProtect = require('assets/images/protect_data_point.svg')
+const twoFaModalSecu = require('assets/images/niv_secu_point.svg')
```

Comment:
you can use import in order to be coherent (and like that we'll not mix esm & commonjs)

Reply:
It's done 🙂